.github/workflows/android-xml-tidy.yml

```diff
@@ -0,0 +1,21 @@
+name: Android app XML formatting verifier
+on:
+    # Run verifier whenever an Android XML file changes
+    push:
+        paths:
+            - .github/workflows/android-xml-tidy.yml
+            - android/**/*.xml
+    # Run verifier if requested manually from the Actions tab
+    workflow_dispatch:
+jobs:
+    xml-tidy:
+        runs-on: ubuntu-latest
+        steps:
+            # Checkout repository
+            - uses: actions/checkout@v2
+
+            # Check formatting
+            - run: |
+                sudo apt-get install tidy
```

Comment:
Will it download each time from remote repo or from the mirror? Will it cost for downloading from remote repo?